wiki/Skinning/osu!/en.md

```diff
@@ -757,5 +757,27 @@ Notes:
 - `hit300.png` must be skinned.
 - This element is used for `hit300`, `hit300g`, and `hit300k`.
 
+## Slider miss indicators (Lazer only)
+
+In the [Lazer](/wiki/Client/Release_stream/Lazer) version of the game, new indicators were added when missing parts of sliders. The following legacy skin elements can be used to skin these. They have no effect in osu!stable<!-- TODO link somewhere -->.
+
+::: Infobox
+
+|  |  |
+| :-- | :-- |
+| Skin versions | All |
+| Animatable | ![Yes][true] |
+| Beatmap skinnable | ![Yes][true] <!-- idk if this is actually true right now but they are implemented the same as hit0 so... -clayton --> |
```

Comment:
I think most things are beatmap skinnable these days. There's very few exceptions for gameplay elements, if any.